index.js

```diff
@@ -56,4 +56,6 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
 
+console.log('Tyriar was here');
```

Comment:
Why would you think this is a good idea?

Reply:
+1

Reply:
+3

Reply:
+4

Reply:
+4

Reply:
+5

Reply:
+5

Reply:
my review

Reply:
start review

Reply:
r1